frontend/src/features/workspace/components/CreateFormModal/CreateFormWizardProvider.tsx

```diff
@@ -72,21 +73,25 @@ const useCreateFormWizardContext = (): CreateFormWizardContextReturn => {
 
   const handleCreateStorageModeOrMultirespondentForm = handleSubmit(
     ({ title, responseMode }) => {
-      if (
-        !(
-          responseMode === FormResponseMode.Encrypt ||
-          responseMode === FormResponseMode.Multirespondent
-        )
-      ) {
-        return
+      switch (responseMode) {
```

Comment:
Would recommend to have [exhaustive switch for enums](https://www.notion.so/opengov/Use-exhaustive-switch-for-enums-e273c275a684492d8aef4789b4882906?pvs=4)

Reply:
Thanks! This is a neat ts feature, sorta like rust's exhaustive pattern match binding